include/common/s1ap_structs.h

```diff
@@ -687,6 +688,7 @@ enum eps_nas_mesage_type {
 	DetachAccept = 0x46,
     TauAccept    = 0x49,
     TauReject    = 0x4b,
+	ServiceReject = 0x4e,
```

Comment:
One time we should arrange all header files as per spec numbers and pull all #define once.. Instead of adding on demand. This will simplify our work.

Reply:
TBD. Will open task for it.